controller/app.controller.js

```diff
@@ -39,4 +40,22 @@ const getArticles = (req, res, next) => {
   });
 };
 
-module.exports = { getTopics, getAll, getArticleById, getArticles };
+const getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  selectCommentsByArticleId(article_id)
+    .then((comments) => {
+      if (comments.length === 0) {
```

Comment:
this conditional should be in the model as a Promise.reject once the result has come back :)